### DISCUSSION TITLE
'Spell Checker status' missing from status bar

### DISCUSSION THREAD

**Discussion by fade2gray** [streetsidesoftware/vscode-spell-checker#3866] [Q&A] [answered]
I inadvertently managed to remove the 'Spell Checker status' from the status bar, and can't find any option in the extension's settings, or elsewhere, to return it - uninstalling/reinstalling does not help (this is on a new PC).

Do you have any advice, please?

Update:
I fired up my old PC, which still had v3.0.1 showing either a cogwheel or checkmark in the status bar. Updating to v4.0.21 replaced the cogwheel/checkmark with a pair of braces.

If revert back to v3.0.1, on the new PC, the cogwheel/checkmark returns to the status bar, but if I then update to any version >= v4.0.18, the cogwheel/checkmark disappear but do not get replaced with the braces. I also see these notifications when updating.

![Screenshot 2024-12-01 135505](https://github.com/user-attachments/assets/65150f2a-bc90-4942-b85c-605bda2cc74e)

¯\\_(ツ)_/¯

![Screenshot 2024-12-01 123144](https://github.com/user-attachments/assets/62e84df5-8d1b-4fb3-a524-692eaadc586a)

**Comment by fade2gray** [comment, marked as answer]
OK, so I had managed to uncheck the 'Editor Language Status' from the status bar context menu.

![PonkesolSolanaGIF](https://github.com/user-attachments/assets/46efc225-c8a1-4850-8a31-443c5764c00d)

**Comment by Jason3S** [comment]
@fade2gray,

I'm happy you figured it out.

**Reply by fade2gray** [reply]
Curios though, why the change from the cogwheel/checkmark to the braces?

**Reply by Jason3S** [reply]
The `{}` braces are part of VS Code. They prefer to have the linter extensions use [`LanguageStatusItem`](https://code.visualstudio.com/api/references/vscode-api#LanguageStatusItem).

You can pin the Spell checker to the status bar:
<img width="326" alt="image" src="https://github.com/user-attachments/assets/735d5b79-2af5-4018-8e51-e450826d75ee">
